File: Android/Tejash11_TejashSeth_2024csit1065_2/GettingStarted/InverseOfANumber.java
import java.util.*;
public class Main
{
    public static void main (String[]args)
    {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt ();
        
        int inverse = 0;
        int power = 1;
        while(n>0)
        {
            int digit = n%10;
            n = n/10;
            inverse += power * Math.pow (10, digit-1);
            power++;
        }
        System.out.println(inverse);
    }
}

File: Android/iamAditya-07_AdityaPratapSingh_2024cse1115_2/GettingStarted/InverseOfANumber.cpp
#include <iostream>
#include <math.h>
using namespace std;

int main(){
  int n;
  cin>>n;
  
  int inverted=0;
  int power=1;
  
  while(n>0){
      int digit=n%10;
      n=n/10;
      inverted+=power*pow(10,digit-1);
      power++;
  }
  cout<<inverted;
  cout<<"\n";
    
}